infra/base-images/base-builder-new/test_new/__init__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2021 Google LLC
```

Comment:
I don't think this existed in base-builder. Why does it need to exist here?

Reply:
I think I can remove it now that the module name is different but before it was because detect_repo and bisect_clang had the same name in the new base-builder and old (so pycache on CI didn't like that)